contrib/bench/build-kernel.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+pushd /home/jolsa/bpf-next-1/tools/perf/
```

Comment:
This feels too specific to your setup, I think :)
How about adding an argument or an env var for the kernel dir?

Reply:
so the idea of this script is to be provided by user and executed as workload with:
  # ./contrib/bench/bench.sh <workload>

I just wanted to have it in, so I don't need to save it elsewhere,
I can put to comment that it's user specific

Reply:
OK, sounds good to me, thanks!
One suggestion would be to just replace /home/jolsa with $HOME to make it somewhat less user-specific.